.travis.yml

```diff
@@ -43,7 +43,9 @@ matrix:
     dist: precise
   - language: node_js
     env: WP_TRAVISCI="yarn lint"
-  - language: node_js
+  # This test requires building Jetpack, so skip on built version branches.
+  - if: branch !~ /(^branch-.*-built)/
```

Comment:
there is an `branches: except:` available in travis that we could use instead? https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches

Reply:
No real preference, don't know if that makes any more or less sense :)

Reply:
My understanding is `branches: except:` would exclude all tests from running on them, which isn't the goal.

Reply:
ah ok, thanks for clarifying that.